pkg/operator/ceph/csi/controller.go

```diff
@@ -205,7 +205,7 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to list ceph clusters")
 	}
 
-	// // Do not nothing if no ceph cluster is present
+	// Do not nothing if no ceph cluster is present
```

Comment:
Not your bug, but good to fix at the same time
```suggestion
	// Do nothing if no ceph cluster is present
```